.github/workflows/workflow_inference.yml

```diff
@@ -106,12 +111,16 @@ jobs:
           TARGET=${{steps.target.outputs.target}}
           if [[ ${{ matrix.model }} == "mpt-7b-bigdl" ]]; then
             docker exec "${TARGET}" bash -c "python inference/serve.py --config_file inference/models/bigdl/mpt-7b-bigdl.yaml --simple"
+          elif [[ ${{ matrix.model }} == "llama-2-7b-chat-hf-vllm" ]]; then
+            docker exec "${TARGET}" bash -c "python inference/serve.py --config_file .github/workflows/config/llama-2-7b-chat-hf-vllm-fp32.yaml --simple"
```

Comment:
If you change llama-2-7b-chat-hf-vllm-fp32.yaml to llama-2-7b-chat-hf-vllm.yaml, you don't need this condition branch. It's better to keep file name and model name same.

Reply:
The config file is different from the one in model/vllm which is BF16 enabled. In the CI case FP32 is enabled instead. It works for now. I will propose to refactor workflow_inference.yml in #63.